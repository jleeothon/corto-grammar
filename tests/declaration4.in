map{string, int32} m
