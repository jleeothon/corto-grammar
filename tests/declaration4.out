(program (statement (declaration (typeLabel (anonymousType (identifier map) (initializerBraces { (initializerList (initializerElem (initializerValue (identifier string))) , (initializerElem (initializerValue (identifier int32)))) }))) m) \n))
